command/acl/token/clone/token_clone.go

```diff
@@ -47,16 +49,26 @@ func (c *cmd) init() {
 	flags.Merge(c.flags, c.http.ServerFlags())
 	flags.Merge(c.flags, c.http.MultiTenancyFlags())
 	c.help = flags.Usage(help, c.flags)
+
+	// Deprecations
+	c.flags.StringVar(&c.tokenID, "id", "",
+		"DEPRECATED. Use -accessor-id instead.")
 }
 
 func (c *cmd) Run(args []string) int {
 	if err := c.flags.Parse(args); err != nil {
 		return 1
 	}
 
-	if c.tokenID == "" {
-		c.UI.Error(fmt.Sprintf("Cannot update a token without specifying the -id parameter"))
-		return 1
+	tokenAccessor := c.tokenAccessorID
+	if tokenAccessor == "" {
+		if c.tokenID == "" {
+			c.UI.Error("Cannot update a token without specifying the -accessor-id parameter")
+			return 1
+		} else {
+			tokenAccessor = c.tokenID
+			c.UI.Warn("Use the -accessor-id parameter to specify token by Accessor ID.")
```

Comment:
minor: I feel like the warning should include what they've done "wrong" (i.e. they used the deprecated flag).

```suggestion
			c.UI.Warn("The -id parameter is deprecated. Use the -accessor-id parameter instead.")
```

(commenting once for all locations with the same warning)

Reply:
Agree with Paul. The current wording also suggests that the operation didn't actually go through (because we're just saying "Do this differently"). Stating that a param is deprecated implies that the operation still went through.